egs/madcat_ar/v1/local/prepare_lang.sh

```diff
@@ -0,0 +1,521 @@
+#!/bin/bash
```

Comment:
How is this script different from utils/prepare_lang.sh?

Reply:
It was used for smaller punctuation topology for Arabic data. But since chainali code used to take around 7-9 days. I was not able to test it. I removed it.